src/merkle/mmr/partial.rs

```diff
@@ -198,7 +271,7 @@ impl PartialMmr {
     /// that element up to its corresponding Mmr peak. The `node` is only used to compute the root
     /// from the authentication path to valid the data, only the authentication data is saved in
     /// the structure. If the value is required it should be stored out-of-band.
-    pub fn add(
+    pub fn track(
         &mut self,
         index: usize,
         node: RpoDigest,
```

Comment:
nit: should we rename `node` to `leaf` and `index` to `leaf_pos` to be consistent with the terminology we use in `add()` and `untrack()`?

Reply:
changed